timesketch/api/v1/resources/settings.py

```diff
@@ -27,46 +15,78 @@ class SystemSettingsResource(Resource):
     @login_required
     def get(self):
         """GET system settings.
-
         Returns:
             JSON object with system settings.
         """
-        # Settings from timesketch.conf to expose to the frontend clients.
-        settings_to_return = ["DFIQ_ENABLED"]
-        result = {}
-
-        for setting in settings_to_return:
-            result[setting] = current_app.config.get(setting)
+        # Settings from timesketch.conf to expose to the frontend clients
+        result = {
+            "DFIQ_ENABLED": current_app.config.get("DFIQ_ENABLED"),
```

Comment:
Not your change, but we should always have a default for settings.

```suggestion
            "DFIQ_ENABLED": current_app.config.get("DFIQ_ENABLED", False),
```

Reply:
Done